docs/UserGuide.md

```diff
@@ -49,18 +49,20 @@ If you can type fast, UGTeach can get your contact management tasks done **faste
 
 ## Features
 
-<box type="info" seamless>**Notes about the command format:**<br>
+<box type="info"><md>**Notes about the command format:**</md><br>
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
   e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
 
 * Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+  e.g `n/NAME [owe/OWED_AMOUNT]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
 
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 
-* No two students can have both same **NAME** and **PHONE**.
+* No two students can have both same **NAME** and **PHONE**, but different students may share a **PHONE** number.
```

Comment:
the parameters provided is PHONE_NUMBER in this case, should we just standardize to just refer to **PHONE_NUMBER**